joss/paper.md

```diff
@@ -29,14 +29,14 @@ bibliography: paper.bib
 ---
 
 # Summary
-Across scientific fields, the Bayesian framework is used to account for uncertainties in inference [@ellison2004bayesian; @lancaster2004introduction; @von2011bayesian]. However, for models with more than a few parameters, exact inference is intractable. A frequently used strategy is to approximately sample the posterior distribution on models' parameters with a Markov chain Monte Carlo (MCMC). Yet conventional MCMC methods relying on local updates can take a prohibitive time to converge when posterior distributions have complex geometries (see e.g., [@rubinstein_simulation_2017] for a textbook).
+Across scientific fields, the Bayesian framework is used to account for uncertainties in inference [@ellison2004bayesian; @lancaster2004introduction; @von2011bayesian]. However, for models with more than a few parameters, exact inference is intractable. A frequently used strategy is to approximately sample the posterior distribution on a model's parameters with a Markov chain Monte Carlo (MCMC) method. Yet conventional MCMC methods relying on local updates can take a prohibitive time to converge when posterior distributions have complex geometries (see e.g., @rubinstein_simulation_2017).
 
-`flowMC` is a Python library implementing accelerated MCMC leveraging deep generative modelling as proposed by @Gabrie2022, built on top of the machine learning libraries JAX [@jax2018github] and Flax [@flax2020github]. At its core, `flowMC` uses a combination of Metropolis-Hastings Markov kernels using local and global proposed moves. While multiple chains are run using local-update Markov kernels to generate approximate samples over the region of interest in the target parameter space, these samples are used to train a normalizing flow (NF) model to approximate the samples' density. The NF is then used in the independent Metropois-Hastings kernel to propose global jumps across the parameter space. The `flowMC` sampler can handle non-trivial geometry, such as multimodal distributions and distributions with local correlations. 
+`flowMC` is a Python library implementing accelerated MCMC leveraging deep generative modelling as proposed by @Gabrie2022, built on top of the machine learning libraries JAX [@jax2018github] and Flax [@flax2020github]. At its core, `flowMC` uses a combination of Metropolis-Hastings Markov kernels using local and global proposed moves. While multiple chains are run using local-update Markov kernels to generate approximate samples over the region of interest in the target parameter space, these samples are used to train a normalizing flow (NF) model to approximate the samples' density. The NF is then used in an independent Metropolis-Hastings kernel to propose global jumps across the parameter space. The `flowMC` sampler can handle non-trivial geometry, such as multimodal distributions and distributions with local correlations. 
```

Comment:
Typo in spelling of Metropolis and I think 'an' is the correct determiner here as there are multiple possible independent Metropolis-Hastings kernels and this is a single instance.